tests/callbacks/test_timer.py

```diff
@@ -106,9 +106,12 @@ def test_timer_stops_training(tmpdir):
         max_epochs=1000,
         callbacks=[timer],
     )
-    trainer.fit(model)
+    with caplog.at_level(logging.INFO):
+        trainer.fit(model)
     assert trainer.global_step > 1
     assert trainer.current_epoch < 999
+    assert "Time limit reached." in caplog.text
+    assert "Signaling Trainer to stop." in caplog.text
```

Comment:
```suggestion
    assert re.match(r"Time limit reached. Elapsed time is .*\. Signaling Trainer to stop.", caplog.text)
```

Reply:
@justusschock mind rather use reg expresions?

Reply:
@awaelchli ^^

Reply:
@Borda it wouldn't work.     

Not even 

`assert re.match(r".*Time limit reached.*", caplog.text)`

wants to match.

Reply:
```python
import re
m = "Time limit reached. Elapsed time is any text here I want. Signaling Trainer to stop."
assert re.match(r"Time limit reached. Elapsed time is .*\. Signaling Trainer to stop.", m)
```
or what is in the `caplog.text`?